src/webrtc_session.ts

```diff
@@ -425,16 +423,16 @@ export class PeerbookSession extends WebRTCSession {
             const offer = btoa(JSON.stringify(d))
             this.pc.setLocalDescription(d)
             terminal7.log("got offer", offer)
-            if (!terminal7.pb)
+            if (!this.t7.pb)
                 console.log("no peerbook")
             else
-                terminal7.pb.send({target: this.fp, offer: offer})
+                this.t7.pb.send({target: this.fp, offer: offer})
```

Comment:
ditto

Reply:
I don't understand what you mean here. `this.t7` is set to the global `terminal7` in the `Session` https://github.com/tuzig/terminal7/blob/master/src/session.ts#L83 so I'm using the local reference

Reply:
the code has many references to `this.t7` and many to a global `terminal7`. It's my fault as I've added the property without thinking it through and now I'm trying to go back to the simpler global as it's a single page web app and will always be.